qutip/tests/test_correlation.py

```diff
@@ -251,3 +253,64 @@ def test_hamiltonian_order_unimportant():
     backwards = qutip.correlation_2op_2t(H[::-1], start, times, times, [sp],
                                          sp.dag(), sp)
     np.testing.assert_allclose(forwards, backwards, atol=1e-6)
+
+
+# Paramters for the test
+solver_list = ['me', 'es']
+start_list = [ 'coherent_dm', 'fock', None]
+e_op_list = ['non-hermitian', 'hermitian']
+w_list = [1,2] # Fast and slow oscillations
+gamma_list = [1,10] # Small and high damping
+args = list(product(solver_list, start_list, e_op_list, w_list, gamma_list))
+
+# We want to mark mc as slow
+solver_list = ['mc']
+start_list = [ 'fock']
+e_op_list = ['non-hermitian', 'hermitian']
+w_list = [1,2] # Fast and slow oscillations
+gamma_list = [1,10] # Small and high damping
+args_slow = list(product(solver_list, start_list, e_op_list, w_list, gamma_list))
+args += [pytest.param(*arg, marks = pytest.mark.slow) for arg in args_slow]
+
+
+@pytest.mark.filterwarnings("ignore::FutureWarning")
+@pytest.mark.parametrize(["solver", "start", "e_op", "w", "gamma"], args)
+def test_correlation_2op_1t(solver, start, e_op, w, gamma):
+    """This test compares the output correlation_2op_1 solution to an analytical
+    solution."""
+
+    a = qutip.destroy(_equivalence_dimension)
+    x = (a + a.dag())/np.sqrt(2)
+
+    H = w * a.dag() * a
+
+    a_op = x if e_op=="hermitian" else a
+    b_op = x if e_op=="hermitian" else a.dag()
+    c_ops = [np.sqrt(gamma) * a]
+
+    if start=='fock':
+        psi0 = _equivalence_fock
+    elif start=='coherent_dm':
+        psi0 = _equivalence_coherent
+    else:
+        psi0 = start
```

Comment:
You can just pass these as the function arguments - see above how I used `pytest.param` to set a non-standard id for them.  Good thinking on trying to keep the tests looking nice, though.